### PR TITLE
019 form

### PR DESCRIPTION
**Pull request by Gonaline**

### CODE REVIEW

File: frontend/src/pages/Contact/index.jsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+//     <p>03 27 21 28 14</p>
+//     <p className="schedules">
+//       Ouvert : Lundi 9:00 - 12:00, 14:00 - 17:00 Mardi 9:00 - 12:00, 14:00
+//       - 17:00 Mercredi 9:00 - 12:00, 14:00 - 17:00 Jeudi 9:00 - 12:00,
+//       14:00 - 17:00 Vendredi 9:00 - 12:00, 14:00 - 17:00
+//     </p>
+//     <div className="map">map</div>
+//   </div>
+// </div>
+// </SContact>
```
Comment (FrancoisSalmon) [inline frontend/src/pages/Contact/index.jsx:44]: comented code

**Review by FrancoisSalmon — CHANGES_REQUESTED** [review]
remove the comented code

**Review by FrancoisSalmon — DISMISSED** [review]
ok

File: frontend/src/pages/Contact/index.jsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+// import Title from "@components/Archi/Title";
```
Comment (LuigiMenez) [inline frontend/src/pages/Contact/index.jsx:1]: No dead code

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
No dead code

**Review by FrancoisSalmon — DISMISSED** [review]
dead code remove

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
Fix conflict

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**